recipes/fft/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class FftConan(ConanFile):
+    name = "fft"
+    license = "LicenseRef-LICENSE"
```

Comment:
What is this? 🙈

Reply:
from https://softwareengineering.stackexchange.com/questions/285885/which-spdx-license-is-equivalent-to-all-rights-reserved
if you have a better idea, suggest :)

Reply:
Thanks for the link

Yes, I think this is probably one of the best solutions `LicenseRef-` as a prefix, followed by the name of the file which contains a custom license for which there is no SPDX entry.

Reply:
I still ask myself, why create a new weird license if you have a bunch of well maintained open source licenses :disappointed: